### ISSUE TITLE
Upgrate to ynh9 from ynh2 broken

### ISSUE THREAD

**Issue by curtosis** [YunoHost-Apps/wallabag2_ynh#227]
### Describe the bug

Running the app upgrade from the web admin or CLI fails with `mysqldump: command not found`. 

### Context

- Hardware: VPS bought online
- YunoHost version: 12.0.9
- I have access to my server: Through SSH | through the webadmin 
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: no
  - If yes, please explain:
- Using, or trying to install package version/branch: 2.5.4~ynh9
- If upgrading, current package version: 2.5.4~ynh2

### Steps to reproduce

```sh
    sudo yunohost app upgrade wallabag2
```

### Logs

[logs here](https://paste.yunohost.org/raw/ovujugociv)

**Comment by curtosis** [comment]
Obviously, upgrade was possible without a backup using `-b` but still thought it should be flagged here.

**Comment by lapineige** [comment]
```
DEBUG - + ynh_mysql_dump_db --database=wallabag2
INFO - Backing up the MySQL database...
DEBUG - + mysqldump --single-transaction --skip-dump-date --routines wallabag2
WARNING - /usr/share/yunohost/helpers.v1.d/mysql: line 154: mysqldump: command not found
```

Wtf is happening… :thinking:

**Comment by lapineige** [comment]
#228
